examples/40_advanced/example_pandas_train_test.py

```diff
@@ -1,26 +1,24 @@
 # -*- encoding: utf-8 -*-
 """
 ===============================
-Test and Train data with Pandas
+Performance-over-time plot
 ===============================
+This example shows, how to use the *performance_over_time_* attribute to plot the performance
+over train time.  *performance_over_time_* **can** contain multiple metrics, namely:
+- ensemble_optimization_score
+- ensemble_test_score
+- single_best_optimization_score
+- single_best_test_score
+- single_best_train_score
 
-*auto-sklearn* can automatically encode categorical columns using a label/ordinal encoder.
-This example highlights how to properly set the dtype in a DataFrame for this to happen,
-and showcase how to input also testing data to autosklearn.
 The X_train/y_train arguments to the fit function will be used to fit the scikit-learn model,
```

Comment:
Auto-sklearn fits both scikit-learn models and an ensemble made from those models. 

I think it makes sense to leave the previous message so that the users have this in mind?